java-strings-ops-2/src/main/java/com/baeldung/string/Anagram.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.string;
+
+import java.util.Arrays;
+import com.google.common.collect.HashMultiset;
+import com.google.common.collect.Multiset;
+
+public class Anagram {
+    //this definition only works for single byte encoding character set.
+    //For multibyte encoding, such as UTF-8, 16, 32 etc., 
+    //we need to increase this number so that it can contain all possible characters.
+    private static int MAX_CHARACTER = 256;
```

Comment:
`Minor` - this isn't the max character, it's the number of characters. MAX=255, MIN=0. `CHARACTER_RANGE` might be more correct. Don't change it if you disagree.